modules/prebidServerBidAdapter/index.js

```diff
@@ -480,17 +477,22 @@ const OPEN_RTB_PROTOCOL = {
       // a seatbid object contains a `bid` array and a `seat` string
       response.seatbid.forEach(seatbid => {
         (seatbid.bid || []).forEach(bid => {
-          const bidRequest = utils.getBidRequest(
-            this.bidMap[`${bid.impid}${seatbid.seat}`].bid_id,
-            bidderRequests
-          );
+          let bidRequest;
+          let key = `${bid.impid}${seatbid.seat}`;
+          if (bidIdMap[key]) {
+            bidRequest = utils.getBidRequest(
+              bidIdMap[key],
+              bidderRequests
+            );
+          }
 
           const cpm = bid.price;
           const status = cpm !== 0 ? STATUS.GOOD : STATUS.NO_BID;
-          let bidObject = bidfactory.createBid(status, bidRequest);
+          let bidObject = bidfactory.createBid(status, bidRequest || {
+            bidder: seatbid.seat,
+            src: TYPE
```

Comment:
I'm not sure if this an issue - but the `src` isn't defined in the `bidRequest` prior to this point.  So when the `bidObject` gets created, the `bidObject.source` is set to `client` and that's carried through the rest of the auction.

Previously it would have been set to `s2s` to reflect it came from pbs via the override.

Reply:
That's a good catch.  It seems to me that the src should be known at that point though as it was already set on the `bidderRequest.src`, just not each of the `bidderRequest.bids` objects.  I updated the pull-request to include adding the `src` to the bids as well as soon as src is known.